setup/install_gatling/30_classification/60_classify_use_loadbalancer.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require File.expand_path("../../../setup/helpers/perf_helper", __dir__)
+Beaker::TestCase.class_eval { include PerfHelper }
+
+test_name "classify and use loadbalancer" do # rubocop:disable Metrics/BlockLength
+  skip_test "Not a PE Large/XL Architecture" unless any_hosts_as?(:loadbalancer) && any_hosts_as?(:compile_master)
+
+  step "install puppet on loadbalancer" do
+    cmd = frictionless_agent_installer_cmd(loadbalancer, {}, "foobar")
+    on(loadbalancer, cmd)
+  end
+
+  step "create loadbalancer groups" do
+    add_loadbalancer_groups(loadbalancer, compile_master)
+  end
+
+  step "add loadbalancer to PE Infra Agent group" do
+    group_id = classifier.get_node_group_id_by_name("PE Infrastructure Agent")
+    classifier.pin_nodes(group_id, { nodes: [loadbalancer] }.to_json)
+  end
+
+  step "export haproxy balancer member resources" do
+    on(compile_master,
```

Comment:
Does this work is there is more than one host with the role compile_master?

Reply:
Yes, beaker commands take host objects or arrays of host objects.